dashboard/src/components/InstanceListView/InstanceListView.tsx

```diff
@@ -194,3 +194,5 @@ export class InstanceListView extends React.PureComponent<
     });
   };
 }
+
+export default InstanceListView;
```

Comment:
Also, a suggestion, "instance" is a bit generic. Would it better to rename it to `ServiceInstance`? The same occurs with `Class` a better name would be `ServiceClass` (or `ClusterServiceClass` to be purist).

Reply:
There is a todo item to rename InstanceListView to InstanceList (both the components and containers). Indeed I did change ServiceCatalogView to ServiceCatalog but I also want to change that one to ServiceBrokerList since that's what it really is. Let's tackle them in separate PRs as you suggested?

Reply:
sure, what do you thing then about renaming `s/InstanceList/ServiceInstanceList` and `s/Class/ServiceClass`?

Reply:
Yes, agreed - we should rename all these components/containers to add a Service prefix. Should I do that here?

Reply:
👍  whatever you prefer, it's okay if you want to do that in a different PR